### PR TITLE
fix: Azerothcore Update some methods to UpperCamelCase

### PR DESCRIPTION
**Pull request by Kitzunu**

### CODE REVIEW

File: UnitMethods.h [hunk 1]
```diff
@@ -838,6 +838,8 @@ namespace LuaUnit
     {
 #ifdef TRINITY
         Eluna::Push(L, unit->GetLevel());
+#elif AZEROTHCORE
+        Eluna::Push(L, unit->GetLevel());
```
Comment (Winfidonarleyan) [inline UnitMethods.h:842]:
```suggestion
#if defined TRINITY || AZEROTHCORE
        Eluna::Push(L, unit->GetLevel());
```

Comment (Winfidonarleyan) [reply UnitMethods.h]: Just replace all

**Review by Winfidonarleyan — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@Rochet2 can you please merge this?

**Comment by Rochet2** [comment]
It probably breaks TC build. So would rather get it fixed before merge.
See https://github.com/ElunaLuaEngine/Eluna/runs/3129279863#step:6:563

**Pull request closed without merging**